oneview/data_source_server_profile.go

```diff
@@ -1116,7 +1120,6 @@ func dataSourceServerProfileRead(d *schema.ResourceData, meta interface{}) error
 			// Get Boot targets list
 			targets := make([]map[string]interface{}, 0)
 			if len(connection.Boot.Targets) != 0 {
-				targets := make([]map[string]interface{}, 0, len(connection.Boot.Targets))
```

Comment:
did you remove this line intentionally..? any reasons for that

Reply:
No, reasssinging the variable inside the if condition will make in inaccessible outside the condition.